### DISCUSSION TITLE
Where can I find the version number in-app?

### DISCUSSION THREAD

**Discussion by 232344235342** [0perationPrivacy/VoIP#44] [Q&A] [answered]
I want to confirm my app is actually up to date (it is according to Heroku and Github) however, I want to check in the web-app to make sure. Apparently this feature has already been added, however, I can't find the version number anywhere? Does this mean my app is glitched? https://github.com/0perationPrivacy/VoIP/projects/1#card-68320060

**Comment by bioscopic** [comment]
I think its all just marked "beta" for now while the bugs are being worked out and there is no version quite yet... Could be wrong i'm sure @0perationPrivacy will tell us...

**Comment by 0perationPrivacy** [comment, marked as answer]
The app version is in the login page at the bottom.
You'll need to logout to see it. In a future update I'll put it somewhere in the settings page. But that settings page will go through a complete re-do once more settings are added. So for now it's on the login page only.

**Comment by 0perationPrivacy** [comment]
Now version number can be found in the login page and the settings menu.
New update notifications compare the local version.md file with the one on the github repo.
